library/core/src/intrinsics/simd.rs

```diff
@@ -209,7 +209,7 @@ pub unsafe fn simd_ne<T, U>(_x: T, _y: T) -> U;
 
 /// Tests if `x` is less than `y`, elementwise.
 ///
-/// `T` must be a vector of floating-point primitive types.
+/// `T` must be a vector of integer or floating-point primitive types.
```

Comment:
"integer or"... mmh, why not just this?

```suggestion
/// `T` must be a vector of integers or floats.
```

The "integer or floating-point primitive types" causes my brain to trip slightly on the "or", since it expects "integer" to be "integers".

Reply:
I agree, it's fixed now. turns out the "primitive types" phrasing was used for `simd_cast` and `simd_as` too, which I've also changed.